admin_apps/partner/partner_utils.py

```diff
@@ -24,13 +24,19 @@ def set_partner_instructions() -> None:
     """
 
     if st.session_state.get("partner_tool", None):
-        if st.session_state["partner_tool"] == "dbt":
-            from admin_apps.partner.dbt import DBT_IMAGE, DBT_INSTRUCTIONS
+        if st.session_state["partner_tool"] == "dbt - SQL Model":
```

Comment:
not required, but if we feel like these raw strings of the selected partner tool appear in lots of places of the codebase, might be worth defining an enum for them.